src/Microsoft.Android.Build.BaseTasks/MSBuildReferences.projitems

```diff
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Build"                Version="$(MSBuildPackageReferenceVersion)" />
+    <PackageReference Include="Microsoft.Build"                Version="$(MSBuildPackageReferenceVersion)" Condition=" '$(_IncludeMicrosoftBuildPackage)' == 'true' " />
```

Comment:
Should this be applied to all Microsoft.Build.* PackageReferences?

Reply:
We could add different properties for each `PackageReference` if we have a use case for them.  The use case we are currently looking at is we want to add a reference to all of the `Microsoft.Build.*` packages _except_ this one.